configure.ac

```diff
@@ -1157,7 +1163,9 @@ m4_ifdef([AS_VAR_IF],[
     TOR_CHECK_CFLAGS(-Wstack-protector)
     TOR_CHECK_CFLAGS(--param ssp-buffer-size=1)
     if test "$bwin32" = "false" && test "$enable_libfuzzer" != "yes" && test "$enable_oss_fuzz" != "yes"; then
-       TOR_CHECK_CFLAGS(-fPIE)
+       if test "$enable_pic" != "yes"; then
+           TOR_CHECK_CFLAGS(-fPIE)
+       fi
        TOR_CHECK_LDFLAGS(-pie, "$all_ldflags_for_check", "$all_libs_for_check")
```

Comment:
Shouldn't that LDFLAGS check be in the if/fi ?

Also, no LDFLAGS for -fPIC ?

Reply:
No -- I'll document why. It's a little subtle.